doc/apimanager-reference.md

```diff
@@ -102,6 +102,7 @@ Generated using [github-markdown-toc](https://github.com/ekalinin/github-markdow
 | Affinity | `affinity` | [v1.Affinity](https://v1-17.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#affinity-v1-core) | No | `nil` | Affinity is a group of affinity scheduling rules |
 | Tolerations | `tolerations` | \[\][v1.Tolerations](https://v1-17.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#toleration-v1-core) | No | `nil` | Tolerations allow pods to schedule onto nodes with matching taints |
 | Resources | `resources` | [v1.ResourceRequirements](https://v1-17.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#resourcerequirements-v1-core) | No | `nil` | Resources describes the compute resource requirements. Takes precedence over `spec.resourceRequirementsEnabled` with replace behavior |
+| Workers | `workers` | integer | No | auto | Defines the number of worker processes |
```

Comment:
If `auto` it's a possible value then Workers is not really an integer.
The way for users to set it as `auto` would then be telling them "leave this empty if you want auto be used". In that case, would be great to clarify in the doc

Reply:
The value in the `Default value` field has the implicit meaning that the value will be set when the field is empty/unset. No need to state it explicitly. 

Updated "auto" for more meaningful content with link to apicast upstream doc.

Reply:
:+1: